src/models/logic/utxo.ts

```diff
@@ -309,31 +310,55 @@ export function createUTXOEvaluatedKey(utxo: UTXOAttribute): string {
     ]);
 }
 
+export function createAggsUTXOEvaluatedKey(
+    aggs: AggsUTXOAttribute,
+    order: "assetType" | "address"
+): string {
+    return JSON.stringify([
+        order === "assetType" ? aggs.assetType : aggs.address
```

Comment:
Doesn't it have to be `order === "assetType" ? aggs.address : aggs.assetType`?

Reply:
The `order` means how to order rows.
When `order` === `asset`, Maybe a user queries Txs using `address`.